### PR TITLE
ReactWeek2-Homework

### PR DESCRIPTION
**Pull request by hemanth06912**

### CODE REVIEW

**Comment by senner008** [comment]
Hi :)
Please start by fixing the 2 things I wrote. Then I'll have a look again

File: react/week2/App.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    />
+  ));
+
+  const addTodo = text => {
+    const newTodos = [...todos, { text }];
+    setTodos(newTodos);
+  };
+
+  useEffect(() => {
+    setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (hemanth06912) [reply react/week2/App.js]: I think I need to use clearTimeout function to clear the setTimeout interval counter berserk issue

File: react/week2/App.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from "react";
```
[the remark this answers is not in the document]
Comment (hemanth06912) [reply react/week2/App.js]: I split my code into four different components

**Comment by senner008** [comment]
I'm still not able to run the application unless I move the files. Does it work on your end?

**Comment by hemanth06912** [comment]
> I'm still not able to run the application unless I move the files. Does it work on your end?

yes, it is working on my end

**Comment by senner007** [comment]
I get all the files in one folder :
See image 

image

**Comment by hemanth06912** [comment]
> I get all the files in one folder :
> See image
> 
> image



> I get all the files in one folder :
> See image
> 
> image

Actually, I am not able to create a pull request through git bash, so I am upload the files and creating the pull request from my account so I cannot upload direct folders like src , but only the files. so that is why it not clear with my files. sorry for my un-ordered files.
I will love to get some help on my github issue on sunday.

**Comment by senner007** [comment]
Ok. Lets talk sunday

**Pull request merged**